src/scripts/main.js

```diff
@@ -1 +1,71 @@
 'use strict';
+
+const bodyElement = document.querySelector('body');
+
+bodyElement.addEventListener('contextmenu', (action) => {
+  action.preventDefault();
+});
+
+// Messages classes-------------------------------------------------------------
+const messageClass = 'message';
+const secondMessageClass = 'message second-message';
+const thirdMessageClass = 'message third-message';
+const errorMessageClass = 'message error-message';
+
+// Messages types---------------------------------------------------------------
+const resolvedMessage = 'First promise was resolved';
+const rejectedMessage = 'First promise was rejected';
+const secondResolveMessage = 'Second promise was resolved';
+const thirdPromiseMessage = 'Third promise was resolved';
```

Comment:
You can combine it in helper function that returns string `${...} promise was ${...}`
And same do with messageClass variables